src/SDKs/HDInsight/DataPlane/Microsoft.Azure.HDInsight.Job/Microsoft.Azure.HDInsight.Job.csproj

```diff
@@ -0,0 +1,25 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+	<!-- Please do not move/edit code below this line -->
+	<Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.reference.props'))" />
+	<!-- Please do not move/edit code below this line -->
+
+	<PropertyGroup>
+		<PackageId>Microsoft.Azure.HDInsight.Job</PackageId>
+		<Description>Azure HDInsight Job SDK Library</Description>
+		<AssemblyName>Microsoft.Azure.HDInsight.Job</AssemblyName>
+		<Version>3.0.0-preview</Version>
+		<PackageTags>Microsoft Azure HDInsight Job;HDInsight;HDInsight Job</PackageTags>
+
+	</PropertyGroup>
+	<PropertyGroup>
+		<TargetFrameworks>net452;netstandard1.4</TargetFrameworks>
+	</PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="WindowsAzure.Storage" Version="8.1.4" />
+  </ItemGroup>
+
+	<!-- Please do not move/edit code below this line -->
```

Comment:
nit: fix indentation

Reply:
@dsgouda Thanks, I have fixed this issue.

Reply:
Force pushed a new commit to put commits together.